bin/worker.ml

```diff
@@ -130,18 +133,35 @@ let state_dir =
     ~docv:"PATH"
     ["state-dir"]
 
+let additional_metric_conv =
+  let of_string s = match Astring.String.cut ~sep:":" s with
+    | Some (name, uri) -> Ok (name, Uri.of_string uri)
+    | None -> Error (`Msg ("Malformed additional metric " ^ s))
+  in
+  let pp ppf (name, uri) = Fmt.pf ppf "%s:%a" name Uri.pp uri in
+  Arg.conv (of_string, pp)
+
+let additional_metrics =
+  Arg.value @@
+  Arg.opt Arg.(list additional_metric_conv) [] @@
+  Arg.info
+    ~doc:"Additional prometheus endpoints to scrape in the form <name>:<uri> \
+    presented as a comma separated list."
```

Comment:
So this would be used to point to other local metrics collectors and re-export those metrics over capnp, and the name:uri would match the `metrics` name configured on the scheduler and the URI is where on the localhost to connect (It could reach out to elsewhere given it's a URI).

Reply:
Exactly. We would run Clarke at `localhost:9090/metrics` and then add to ocluster-worker `--additional-metrics=clarke:http://localhost:9090/metrics`. 

Good point re:URI, if this isn't a useful feature we could perhaps limit to a `<name>:<port>` mapping and just always combine that into `http://localhost:<port>/metrics`?